travis/test_pylint

```diff
@@ -52,6 +52,21 @@ import travis_helpers
 
 from getaddons import get_modules_changed
 from git_run import GitRun
+from apis import GitLabApi
+
+
+def get_gitlab_environments():
+    if not os.environ.get('GITLAB_CI'):
+        return {}
+    glb = GitLabApi()
+    envars = glb.get_environments() or {}
+    repo_path = os.path.join(os.environ.get('CI_PROJECT_DIR', ''),
+                             '.git')
+    gitrun = GitRun(repo_path)
+    remote = gitrun.run(["remote", "get-url", "origin"])
+    if remote != glb.remote_url:
+        gitrun.run(['remote', 'set-url', 'origin', glb.get_gitlab_remote()])
```

Comment:
@moylop260 

This I did change the remote because gitlab-ci by default brings the branch of development

![remotes](https://user-images.githubusercontent.com/16363083/27187535-7e7e7b02-51b9-11e7-8868-41033a49672d.png)

this link [build](https://git.vauxoo.com/vauxoo-dev/instance/builds/27221)